.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    services:
+      postgres:
+        image: postgres:9.4
+        ports: ["5432:5432"]
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        env:
+          POSTGRES_USER: postgres
```

Comment:
I would rather replace it with `DATABASE_USER` so we don't need to read from an extra ENV var

Reply:
ah ok! was thinking the `postgres` container will take that naming, I gonna change it and try with more `retries` 🤞🏼